### DISCUSSION TITLE
MultiSelect 组件在Table 中不能正常工作，选择时，编辑框内不显示已选择项。

### DISCUSSION THREAD

**Discussion by mswang2005** [dotnetcore/BootstrapBlazor#3139] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

MultiSelect 组件在Table 中不能正常工作，选择时，编辑框内不显示已选择项。
[BootstrapBlazor.OnlyServer1.zip](https://github.com/dotnetcore/BootstrapBlazor/files/14735140/BootstrapBlazor.OnlyServer1.zip)

![error](https://github.com/dotnetcore/BootstrapBlazor/assets/23739744/c64a8c9c-903e-49f5-bd9c-078e4ea26d91)

### .NET Version

.net 8

**Comment by bb-auto** [comment]
@mswang2005 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@mswang2005 

https://github.com/dotnetcore/BootstrapBlazor/issues/3104?wt.mc_id=DT-MVP-5004174

https://www.blazor.zone/table/edit#EditTemplate

https://gitee.com/LongbowEnterprise/BootstrapBlazor/issues/I9B0P9

https://gitee.com/LongbowEnterprise/BootstrapBlazor/issues/I8W8RS

**Comment by ft115637850** [comment]
8.5.0中依然有此问题。

**Reply by ArgoZhang** [reply]
@ft115637850 帮忙点个 star 先，谢谢啦。我试试

**Reply by xxxlihui** [reply]
8.5.11还是有问题

**Reply by ArgoZhang** [reply]
@xxxlihui 右上角帮忙点一下 `star` 没有问题的。是用法不对
